worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -233,7 +233,7 @@ class PushDataHandler(val workerSource: WorkerSource) extends BaseMessageHandler
     // During worker shutdown, worker will return HARD_SPLIT for all existed partition.
     // This should before return exception to make current push data can revive and retry.
     if (shutdown.get()) {
-      logInfo(s"Push data return HARD_SPLIT for shuffle $shuffleKey since worker shutdown.")
+      logDebug(s"Push data return HARD_SPLIT for shuffle $shuffleKey since worker shutdown.")
```

Comment:
This log should be kept info level to let users know, because it is not printed frequently and involves the Worker shutdown.

Reply:
@zwangsheng, when worker shutdown, why user does not know why return `HARD_SPLIT`? BTW, have you used the log in production practice?

Reply:
You're right, the response status corresponding to `HARD SPLIT` has been clearly described in [the documentation](https://github.com/apache/incubator-celeborn/blob/ced6f93bc1174b53ae87827482b1907ae10ee7d5/docs/upgrading.md?plain=1#L62-L73), users can clearly understand the scene at this time.

Reply:
@zwangsheng, therefore should I keep info level?

Reply:
I' fine with change this level to debug, for reducing non important  log.